src/shared/util.js

```diff
@@ -60,6 +60,13 @@ const AnnotationEditorType = {
   INK: 15,
 };
 
+const AnnotationEditorParamsType = {
+  FONT_SIZE: 0,
+  FONT_COLOR: 1,
```

Comment:
For consistency with the names below, should these perhaps be named `FREETEXT_SIZE`/`FREETEXT_COLOR` instead (or am I misunderstanding things here)?